Tests/Packet++Test/main.cpp

```diff
@@ -7,32 +7,30 @@
 #include "Logger.h"
 #include "../../Tests/Packet++Test/Utils/TestUtils.h"
 
-static struct option PacketTestOptions[] =
-{
-	{"include-tags",  required_argument, nullptr, 't'},
-	{"exclude-tags",  required_argument, nullptr, 'x'},
-	{"show-skipped-tests", no_argument, nullptr, 'w' },
-	{"mem-verbose", no_argument, nullptr, 'm' },
-	{"verbose", no_argument, nullptr, 'v' },
-	{"skip-mem-leak-check", no_argument, nullptr, 's' },
-	{"help", no_argument, nullptr, 'h' },
-	{nullptr, 0, nullptr, 0}
+static struct option PacketTestOptions[] = {
+	{ "include-tags",        required_argument, nullptr, 't' },
+	{ "exclude-tags",        required_argument, nullptr, 'x' },
+	{ "show-skipped-tests",  no_argument,       nullptr, 'w' },
+	{ "mem-verbose",         no_argument,       nullptr, 'm' },
+	{ "verbose",             no_argument,       nullptr, 'v' },
+	{ "skip-mem-leak-check", no_argument,       nullptr, 's' },
+	{ "help",	            no_argument,       nullptr, 'h' },
+	{ nullptr,	           0,                 nullptr, 0   }
```

Comment:
clang-format adds redundant tabs, we should probably disable formatting here

Reply:
fixed.